contrib/etckeeper/patches/use_apk.patch

```diff
@@ -0,0 +1,16 @@
+--- etckeeper.conf.orig	2024-05-21 18:34:47.193839464 +0100
++++ etckeeper.conf	2024-05-21 18:35:43.190313880 +0100
```

Comment:
```suggestion
--- a/etckeeper.conf.orig	2024-05-21 18:34:47.193839464 +0100
+++ b/etckeeper.conf	2024-05-21 18:35:43.190313880 +0100
```

these should all be p1 form (bsd patch seems to accept it even when -p1 is passed.. but iirc gnu patch will not)